frontend/src/features/admin-form/responses/ResponsesPage/storage/useDecryptionWorkers.ts

```diff
@@ -181,7 +190,24 @@ const useDecryptionWorkers = ({
                       errorCount++
                       console.error('Error in getResponseInstance', e)
                     }
+
                     if (downloadAttachments && decryptResult.downloadBlob) {
+                      // Ensure attachments downloads are spaced out to avoid browser blocking downloads
+                      if (progress % ATTACHMENT_DOWNLOAD_CONVOY_SIZE === 0) {
+                        const now = new Date().getTime()
+                        const elaspedSinceXDownloads =
```

Comment:
nit: typo in elapsed

Reply:
[e4c1e24](https://github.com/opengovsg/FormSG/pull/7449/commits/e4c1e249bbcc5ed720dc801e4c59f397c7ff1c89)